app/src/main/java/com/limelight/utils/SamsungDexUtils.kt

```diff
@@ -0,0 +1,23 @@
+package com.limelight.utils
+
+import android.app.Activity
+import android.content.ComponentName
+import android.util.Log
+
+object SamsungDexUtils {
+    val TAG = this::class.java.name
+
+    fun dexMetaKeyCapture(activity: Activity) {
+        runCatching {
+            val semWindowManager = Class.forName("com.samsung.android.view.SemWindowManager")
+            val getInstanceMethod = semWindowManager.getMethod("getInstance")
+            val manager = getInstanceMethod.invoke(null)
+
+            val requestMetaKeyEvent = semWindowManager.getDeclaredMethod(
+                    "requestMetaKeyEvent", ComponentName::class.java, Boolean::class.java)
+            requestMetaKeyEvent.invoke(manager, activity.componentName, true)

```

Comment:
Is this technique documented anywhere or did you figure it out yourself?

Reply:
Samsung didn't document that part on their developer website.
I saw Microsoft RDP handle keys on Samsung Dex, do debugging while running Microsoft RDP and figure out those changes.

Reply:
Also , it seems to capture WinKey and Alt + Tab also in Tablet mode, not only in Dex.
Would that changes make it work on other devices, not only Samsung?
I don't have other devices to test it out.

Reply:
> Also , it seems to capture WinKey and Alt + Tab also in Tablet mode, not only in Dex.

Nice! That will be useful for Samsung devices.

> Would that changes make it work on other devices, not only Samsung?

No, `requestMetaKeyEvent()` is not part of AOSP, so probably only Samsung devices support it.